src/popup/components/ad-hoc-tools-panel.tsx

```diff
@@ -30,7 +34,11 @@ export const AdHocToolsPanel = NamedFC<AdHocToolsPanelProps>('AdHocToolsPanel',
             <span key={`divider-${dividerIndex++}`} className={styles.divider}></span>
         );
 
-        const totalRows = 3;
+        let totalRows = 3;
+        if (props.featureFlagStoreData[FeatureFlags.showAccessibleNames]) {
+            totalRows = 4;
+            addNewRow = true;
```

Comment:
Instead of setting `addNewRow` here, you can set a variable to `styles.newRowNeeded` and use that directly in your component later on; would reduce one layer of indirection.